www/js/viewer.js

```diff
@@ -298,21 +298,50 @@ const createCards = (rows, LineID) => {
     gurmukhiContainer.innerHTML = `<span class="padchhed">${padched}</span>
                                     <span class="larivaar">${larivaar}</span>`;
 
-    const englishContainer = document.createElement('div');
-    englishContainer.innerHTML = row.English || '';
+    const enTranslation = h('div.english-translation.transtext');
+    enTranslation.innerHTML = row.English || '';
+    const esTranslation = h('div.spanish-translation.transtext');
+    // If spanish translation is not available fall back to english
```

Comment:
should we do this? cc @maneetpaul

Reply:
I did this specially for the ceremonies where you have full page filled with english (explanations) because it would otherwise show nothing - so it goes to spanish if english is not available.

Reply:
Note: Make it always english.